doc/guide/feature-ovirt.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+        "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-ovirtvirtualmachines">
+    <title>oVirt Virtual Machines</title>
+
+    <para>Cockpit can connect to <emphasis>oVirt REST API</emphasis> to extend
```

Comment:
Can you put in &lt;ulink&gt; tags around initial uses of oVirt and RHV so that people know what they are?

Reply:
Done